data/cmsswdata.txt

```diff
@@ -3,10 +3,10 @@
 #Once a non-default section is empty then cleanup that section and remove its cmsdist/${PACKAGE_TYPE}.file
 #If there is no customization for the packae then remove its .spec and .file
 [default]
+L1Trigger-L1TMuon=V01-06-00
```

Comment:
@perrotta , shouldn't this be `V01-07-00`?

Reply:
> @perrotta , shouldn't this be `V01-07-00`?

Indeed, thank you @smuzaffar